src/uwtools/rocoto.py

```diff
@@ -26,6 +48,67 @@ def _add_jobname(tree: dict) -> None:
 
 
 # Public functions
+def realize_rocoto_xml(
+    input_yaml: str,
```

Comment:
I'm trying to have it both ways - leaving the `write()` function to accept an external config and any input, but making `realize()` the interface that only uses our Rocoto schemas. Honestly it might be a mistake to try for it and just fix everything to Rocoto-only

Reply:
Yeah, I think you're right, I've just changed all of the `input_yaml` calls to `config_file`